### PR TITLE
Persian Scripts 

### PR DESCRIPTION
**Pull request by DhruboSaeed**
Persian Scripts added

### CODE REVIEW

File: code/persian/index.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+'\u0648': ['u','ow','v', 'o', 'u','w','aw', 'oː'],
+'\u0647': ['h','or ','a'],
+'\u06CC': ['j','i','ɒː', 'aj','eː','ɒː']
+}
+
+
+const otherCharacters ={
+'\u0622' : 'ɒ',
+'\u06C0' : 'eje',
+'\u0644, \u0627' : 'lɒ'
```
Comment (lancejpollard) [inline code/persian/index.ts]: This one has a comma in it, it looks like two characters FYI.

**Comment by lancejpollard** [comment]
@DhruboSaeed looks good enough I think!

**Comment by DhruboSaeed** [comment]
thanks boss.

File: code/persian/index.ts (hunk 1 above)
Comment (lancejpollard) [inline code/persian/index.ts]:
```suggestion
  '\u0644': 'lo',
  '\u0627' : 'lɒ' ,
```

I think these should be separated, correct?

File: code/persian/index.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+'\u0647': ['h','or ','a'],
+'\u06CC': ['j','i','ɒː', 'aj','eː','ɒː']
+}
+
+
+const otherCharacters ={
+'\u0622' : 'ɒ',
+'\u06C0' : 'eje',
+'\u0644, \u0627' : 'lɒ' 
+}
```
Comment (lancejpollard) [inline code/persian/index.ts:52]:
```suggestion
}

const symbols = {
  ...consonants,
  ...otherCharacters
}

const s = build(symbols)

const make = (t: string) => transform(t, s, characters)

export default make
```

Add this at the bottom of every file.

File: code/persian/index.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+const consonant = {
```
Comment (lancejpollard) [inline code/persian/index.ts]: Add this to the beginning of each file.

```suggestion
import { build, transform } from '~/base'

const consonant = {
```

**Comment by lancejpollard** [comment]
@DhruboSaeed in addition to my suggestions above, add a test case using the language like [here](https://github.com/termsurf/text.js/blob/make/test/make.test.ts#L9):

```ts
console.log(makePersian('<some persian text>'))
```

Oh also add this map to each file in the future too, so it supports basic punctuation:

```ts
import { basicPunctuation } from '~/base'

const symbols = {
  ...yourOtherStuff,
  ...basePunctuation
}
```

**Review by DhruboSaeed — COMMENTED** [review]
add the suggestions
1. Added top code block
2. Separate the Unicode ID
3. Added bottom section
4.  Did not correct the bottom section code. Need some elaboration in details

File: code/persian/index.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import { build, transform } from '~/base'
+
+const consonants = {
+'\u0621': 'ʔ ',
```
Comment (lancejpollard) [inline code/persian/index.ts]: There are spaces in here, there must be no spaces.

Comment (lancejpollard) [reply code/persian/index.ts]: Also, @DhruboSaeed did you run `node run move:talk` or `pnpm move:talk`?

Comment (lancejpollard) [reply code/persian/index.ts]: Also, please run `pnpm install` on the repo, so you can get the clean on save functionality.

**Comment by DhruboSaeed** [comment]
Boss One help. I am not solving these errors. Please last help. 

Errors

e2

File: code/persian/index.ts (hunk 4 above)
Comment (lancejpollard) [reply code/persian/index.ts]: @DhruboSaeed did you do these steps yet too? (remove whitespace in value string, and run `pnpm install`).

Comment (lancejpollard) [reply code/persian/index.ts]: If you don't have `pnpm` installed, then use:

```
npm install -g pnpm
```

And you will get it!

**Comment by lancejpollard** [comment]
@DhruboSaeed that error saying:

> type `string[]` is not assignable to type `string | ...`

Is saying that an array is being used where a string should be being used (or a function). I added support for an array to be used. Sorry!

File: code/persian/index.ts (hunk 4 above)
Comment (DhruboSaeed) [reply code/persian/index.ts]: I have remove the whitespace in the value string and install the pnpm at repo. 
didnot know why it is showing all red. 
Image1
Imag2

**Pull request merged**